### DISCUSSION TITLE
FAQ clarification, deprecation strategy: What is "in-system", "in-box"?

### DISCUSSION THREAD

**Discussion by riverar** [microsoft/WindowsAppSDK#436] [Q&A] [answered]
The FAQ [has question](https://github.com/microsoft/ProjectReunion/blob/main/docs/faq.md#what-is-the-deprecation-strategy-for-in-system-components):
> What is the deprecation strategy for in-system components?

And answer:
> In-box components will be supported forever in the usual way. 

Can someone clarify what this means? What does `in-system` mean? What does `supported forever in the usual way` mean? Is it forever or not? Is this meant to be a reference to the [Modern Lifecycle Policy](https://docs.microsoft.com/en-us/lifecycle/policies/modern)? Is that different from `in-box`? Is this FAQ entry trying to tell me that features connected to the OS will continue working, using a polyfill where needed, if the OS removes this functionality?

**Comment by jonwis** [comment, marked as answer]
Sorry for the terminology confusion.  "In system" and "in box" here means "shipping with Windows."  For example, `Windows.ApplicationModel.Core.CoreApplication` is an in-box shipped-with-Windows object that's a part of the component supporting system-managed app lifecycle.  While Project Reunion is getting its own `CoreApplication`-alike (see #111 for instance) to support many of those behaviors, the _in windows_ component will continue to work and will _not_ be deprecated.

The intent is to say that nothing in Project Reunion changes any of our existing support policies for Windows. Your existing apps targeting Windows will continue to work and be supported.  If you start building a new Windows app against the Windows SDK tomorrow, that app will continue to work.

With Project Reunion we can deliver platform-level functionality faster than the normal Windows update cycle. Component owners - such as that CoreApplication type - can bring new features to Project Reunion first. Those *new* features may come only to Project Reunion in some cases, as it's a better return on our work to make them available here first for everybody than _also_ put them into Windows and wait.  That - putting features here first - is not _deprecation_ of the Windows type.

Periodically we do deprecate Windows functionality, such as [AudioRenderEffectsManager.ShowSettingsUI](https://docs.microsoft.com/uwp/api/Windows.Media.Effects.AudioRenderEffectsManager.ShowSettingsUI?view=winrt-19041). But we do so over the course of several releases with clear communication to reduce impact on apps that rely on the functionality.

**Reply by riverar** [reply]
Thanks, so I think a better plain-English answer to the FAQ question then becomes something like:
> Microsoft may ship, as part of Project Reunion, different or newer versions of components that also ship with Windows. These components do not deprecate or obsolete the components that ship with Windows.

Sound right?
